[TAXII 2 Feed] Remove default list of objects types (#28710)
demisto/content: Packs/FeedDHS/ReleaseNotes/2_0_18.md

#### Integrations

##### DHS Feed v2

> - Updated the TAXII2ApiModule to fetch all available object types by default unless the *STIX Objects To Fetch* parameter is provided, in which case, type filtering for objects will be applied based on the parameter's value.

Reviewer: ```suggestion
- Updated the TAXII2ApiModule to fetch all available object types by default when the *STIX Objects To Fetch* parameter is not provided.
```
